include/glow/Graph/Graph.h

```diff
@@ -42,6 +42,9 @@ using FunctionList = std::list<Function *>;
 using VariablesList = std::list<Variable *>;
 using UnsignedArrayRef = llvm::ArrayRef<size_t>;
 
+constexpr char intTanhMappingVarName[] = "int_tanh_mapping";
```

Comment:
I'd suggest moving these specific reserved names into the cpp file. We may need more later. No need to enumerate them here and force client to recompile if we change this set.

For the same reason, I'd move the `Module` constructor below into a cpp file as well.